src/Element.js

```diff
@@ -1,5 +1,6 @@
 import Node from './Node';
 import ParentNode from './ParentNode';
+import CssSelectorParser from 'css-selector-parser';
```

Comment:
Oh, tryed several ways and stoped on this. Yep object destruction should be cool here.